### PR TITLE
:beetle: Fix Vive support.

### PR DESCRIPTION
**Pull request by TheJhonny007**
See https://github.com/cosmoscout/vista/pull/10.

### CODE REVIEW

File: make_externals.bat [hunk 1]
```diff
@@ -78,6 +78,8 @@ cmake -E make_directory "%INSTALL_DIR%/share"
 cmake -E make_directory "%INSTALL_DIR%/bin"
 cmake -E make_directory "%INSTALL_DIR%/include"
 
+goto :openvr
+
```
Comment (Schneegans) [inline make_externals.bat:82]:
```suggestion
```

File: make_externals.bat [hunk 2]
```diff
@@ -287,6 +289,24 @@ cmake %CMAKE_FLAGS% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX="%INS
 
 cmake --build . --config %BUILD_TYPE% --target install --parallel %NUMBER_OF_PROCESSORS%
 
+rem OpenVR ----------------------------------------------------------------------------------------
+:openvr
+
+echo.
+echo Building and installing OpenVR ...
```
Comment (Schneegans) [inline make_externals.bat:296]:
```suggestion
echo Installing OpenVR ...
```

File: make_externals.bat [hunk 3]
```diff
@@ -287,6 +289,24 @@ cmake %CMAKE_FLAGS% -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DCMAKE_INSTALL_PREFIX="%INS
...
+echo.
+
+rem cmake -E make_directory "%BUILD_DIR%/openvr" && cd "%BUILD_DIR%/openvr"
+
+rem cmake %CMAKE_FLAGS% -DCMAKE_INSTALL_PREFIX="%INSTALL_DIR%" -DBUILD_SHARED=On^
+rem       -DCMAKE_UNITY_BUILD=%UNITY_BUILD%^
+rem       "%EXTERNALS_DIR%/openvr" || exit /b
+rem cmake --build . --config %BUILD_TYPE% --target install --parallel %NUMBER_OF_PROCESSORS%
```
Comment (Schneegans) [inline make_externals.bat:304]:
```suggestion
```

File: make_externals.bat [hunk 4]
```diff
@@ -303,11 +323,14 @@ rem       -DVISTADRIVERS_BUILD_3DCSPACENAVIGATOR=On^
...
 cmake %CMAKE_FLAGS% -DCMAKE_INSTALL_PREFIX="%INSTALL_DIR%" -DVISTADEMO_ENABLED=Off^
-      -DCMAKE_BUILD_TYPE=%BUILD_TYPE%^
+      -DCMAKE_BUILD_TYPE=%BUILD_TYPE% -DVISTACORELIBS_USE_VIVE=On -DVISTADRIVERS_BUILD_VIVE=On^
+      -DOPENVR_ROOT_DIR=%INSTALL_DIR%^
       -DCMAKE_UNITY_BUILD=%UNITY_BUILD% -DVISTA_USE_PRECOMPILED_HEADERS=%PRECOMPILED_HEADERS%^
       "%EXTERNALS_DIR%/vista" || exit /b
 cmake --build . --config %BUILD_TYPE% --target install --parallel %NUMBER_OF_PROCESSORS%
 
+goto :finish
+
```
Comment (Schneegans) [inline make_externals.bat:333]:
```suggestion
```

**Comment by Schneegans** [comment]
I think that I have a solution for the CI issue. Just takes some time to test...

**Comment by TheJhonny007** [comment]
> Once the CI works, you may merge this. Maybe my commit does the trick.

We got further 😅. I will try to resolve this.

PS: This is one of the reasons for https://github.com/cosmoscout/vista/issues/9

**Comment by TheJhonny007** [comment]
I did make some more changes in Vista: See https://github.com/cosmoscout/vista/pull/11

**Pull request merged**